addons/account/views/account_move_views.xml

```diff
@@ -306,6 +306,7 @@
                     <field name="name" string="Journal Item" filter_domain="[
                         '|', '|', '|',
                         ('name', 'ilike', self), ('ref', 'ilike', self), ('account_id', 'ilike', self), ('partner_id', 'ilike', self)]"/>
+                    <field name="amount_currency" filter_domain="[('amount_currency', '&gt;=', self)]" />
```

Comment:
You used the amount_currency but i don't think that really what we want :eyes: The amount currency will be the same as the balance or debit and credit when we have the same currency but it could happen that you do a transaction in dollars when your company is in euro and so the amount_currency will be different. Here we want to be able to search on the balance or debit and credit together :smile: